cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -38,8 +63,15 @@ namespace cobalt {
 namespace {
 
 constexpr base::FilePath::CharType kCacheDirname[] = FILE_PATH_LITERAL("Cache");
+const char kCobaltExperimentName[] = "CobaltExperiment";
```

Comment:
nit: Prefer `constexpr std::string` for string constants known at compile time.

Reply:
I don't think constexpr std::string can be defined outside of constexpr functions. Unless you're referring to constexpr std::string_view?